Framework: Fix path using platform specific separators

stephanethomas opened the pull request:
This pull request fixes https://github.com/Automattic/wp-calypso/issues/27400 by updating a path that was using platform specific separators, hence generating path errors on Windows.

#### Testing instructions

1. Run `git checkout fix/wrong-pathname` and start your server, or open a [live branch](https://calypso.live/?branch=fix/wrong-pathname)
2. Assert that no error was raised during the build process
3. Assert that everything works as usual

flexpeace commented:
Thanks guys!!!

flexpeace commented:
@stephanethomas I checked out the new branch fix/wrong-pathname and the issue is still persisting. I removed nodejs and reinstalled and still having this issue. I checked your fix and the changes you made are there yet the issue pops up.

tofanelli commented:
still on the same road as @flexpeace .... I will post the screenshots on #27337...

tofanelli commented:
@stephanethomas s ..... still running on issues road.... do I did something at this time??

deleted the folder, restarted the machine (I really don't know why, but did), internet connection fine...
![image](https://user-images.githubusercontent.com/5512922/46039155-69cdbc00-c0e3-11e8-9d1a-19c904410ba9.png)

After few minutes of issues rolling down my screen... still the same issue
![image](https://user-images.githubusercontent.com/5512922/46039166-70f4ca00-c0e3-11e8-837e-155733720dcc.png)

Says its ready to fun, but no =(
![image](https://user-images.githubusercontent.com/5512922/46039180-76521480-c0e3-11e8-937d-9980d0bf5372.png)

This process was done in Administrator mode...

Am I missing some point??

stephanethomas commented:
@flexpeace, @tofanelli: I've spotted a few more paths that weren't correctly handled, and rebased. Could you give this pull request another try?

tofanelli commented:
Hey @stephanethomas ... sorry for late reply... here we go

deleted everything... starting from scratch again
![tela 01](https://user-images.githubusercontent.com/5512922/46233518-4e63ea80-c349-11e8-9c5b-35fe02da4273.jpg)

so far, so good
![image](https://user-images.githubusercontent.com/5512922/46233952-0cd43f00-c34b-11e8-9c6d-cb9e81f50eca.png)

and no, no fun to rock today =\
![image](https://user-images.githubusercontent.com/5512922/46233952-0cd43f00-c34b-11e8-9c6d-cb9e81f50eca.png)

![image](https://user-images.githubusercontent.com/5512922/46234184-d9de7b00-c34b-11e8-8071-82edf0437745.png)


If I open the browser and head to Calypso address, doesn't loads anything...

So, going back to zero again =( .... next step? =)

stephanethomas commented:
I've rebased and updated this pull request according to [@Aurelain's feedback](https://github.com/Automattic/wp-calypso/issues/27400#issuecomment-426410154). However I'm still getting warnings in the Node console as well as the following error in the browser console:

```
Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'
```

highstreetmail commented:
This issue is stubborn. Thanks guys and hope this gets resolved
